sql/core/src/main/scala/org/apache/spark/sql/jdbc/OracleDialect.scala

```diff
@@ -29,7 +29,12 @@ private case object OracleDialect extends JdbcDialect {
   override def getCatalystType(
```

Comment:
I looked over the previous releases though, it seems `scale` always is set there. So, I'm not sure why this exception happens in the report. What do u think? Is it okay to close this?